readme.md

```diff
@@ -0,0 +1,34 @@
+# Emojo
+Lambda function to be used as a slack app to add uploaded emoji to github repo to then be deployed to slack through CI.
+
+### Config
+All config is done through environment variables. These can be set in a `.env` file in the root of the project. See `.env.example`
+
+#### `GITHUB_REPO`
+Name of the repo you wish to save your emoji too
+
+#### `GITHUB_REPO_DIR`
+A sub-directory in the repo where the images themselves will be added.
+
+#### `GITHUB_REPO_BRANCH`
+Custom branch to commit the uploaded emoji to
+
+### Deploy
+Emojo is built using [`serverless`][1] so once that is configured you can run:
+```
+$ sls deploy -v
```

Comment:
There's a yarn command for this. Doesn't rely on global sls then

```suggestion
$ yarn deploy
```

Reply:
noice